### PR TITLE
{phys}[intel/2016b] BAGEL v1.1.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/3b2392cdc98e1173b0134478081e5ad4 for a full test report.

File: easybuild/easyconfigs/b/BAGEL/BAGEL-1.1.1-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+name = 'BAGEL'
+version = '1.1.1'
+
+homepage = "http://www.nubakery.org"
+description = """BAGEL (Brilliantly Advanced General Electronic-structure Lirary)
+is a parallel electronic-structure program."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+# Note: A compiler bug in template deduction prevents newer versions of icpc to compile this software.
```
[the remark this answers is not in the document]
Comment (Micket) [reply easybuild/easyconfigs/b/BAGEL/BAGEL-1.1.1-intel-2016b.eb]: I may have been to hasty to call it a compiler bug. I did a lot more (tedious) and it all boils down to boost_serialization, and is probably related to https://github.com/nubakery/bagel/blob/master/src/util/serialization.h#L104

Only seems to affect the more recent icpc versions though. GCC compiles without errors.
So, I'm not sure if it's a compiler bug, boost-bug, and I have no link.
Poking around and trying different thing in serialization.h didn't help either, so I have no patch (GCC doesn't seem to care whether or not these shared_ptr specializations are present, compiles either way).

But, as bagel doesn't really support OpenBLAS either (https://github.com/nubakery/bagel/issues/116, https://github.com/nubakery/bagel/issues/117). Though, that issue should be resolved in the next release of OpenBLAS and bagel.
I currently are not building any g*mkl toolchains, but those would probably be the best fit for BAGEL.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2435.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/03e34759971a09b0e98174dcfaffd95b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c163a6e05a2369fa6741154773a7700e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**